Rakefile

```diff
@@ -20,6 +20,44 @@ require 'bundler/gem_tasks'
 
 require "rspec/core/rake_task"
 
+require "active_record"
+
+root = File.expand_path(__dir__)
```

Comment:
Perhaps prefer a Pathname here to make the later usage of it cleaner?

Reply:
:+1: yeah that's a lot better